### PR TITLE
readme: Improve markdown, use table

### PR DESCRIPTION
**Pull request by Haxk20**
Based on:
https://github.com/sonyxperiadev/device-sony-edo/commit/2f89cb6fc39fcbce02b0c1382fb34fa9357fd6ee

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
+Yoshino platform
+==============
 Copyright (C) Sony Mobile Communications 2017
-=============================================
+---------------------------------------------
 
-This is the Android device configuration for yoshino platform.
+Android device configuration for the yoshino platform.
```
Comment (MarijnS95) [inline README.md:6]: Don't forget to add the SoC name here (and all other platforms where it was missed - I don't want to comment the same thing on all :wink:).

File: README.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
+Yoshino platform
+==============
```
Comment (MarijnS95) [inline README.md:2]: You might as well use three `===` signs here (and three `---` dashes below), _or_ keep them exactly aligned :wink:

File: README.md (hunk 1 above)
Comment (Haxk20) [reply README.md:6]: yep

File: README.md (hunk 2 above)
Comment (Haxk20) [reply README.md:2]: Perfectionist inside me has to get it alligned. Otherwise my brain will go nuts :)

So i will fix it :)

File: README.md [hunk 3]
```diff
@@ -1,13 +1,18 @@
+Yoshino platform
+================
 Copyright (C) Sony Mobile Communications 2017
-=============================================
+---------------------------------------------
 
-This is the Android device configuration for yoshino platform.
+Android device configuration for the yoshino platform. (**MSM8998**)
```
Comment (MarijnS95) [inline README.md:6]:
```suggestion
Android device configuration for the yoshino platform (**MSM8998**).
```

Just like it was done for Edo. It seems to be correct in Ganges though...

File: README.md [hunk 4]
```diff
@@ -1,13 +1,18 @@
+Yoshino platform
+================
 Copyright (C) Sony Mobile Communications 2017
-=============================================
+---------------------------------------------
 
-This is the Android device configuration for yoshino platform.
+Android device configuration for the yoshino platform (**MSM8998**)
```
Comment (MarijnS95) [inline README.md:6]: Now you removed the period altogether?

Comment (Haxk20) [reply README.md:6]: Im sorry im very tired :(

Comment (Haxk20) [reply README.md]: Fixed

**Review by MarijnS95 — APPROVED** [review]

**Pull request merged**